firecord.gemspec

```diff
@@ -4,33 +4,36 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'firecord/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = "firecord"
+  spec.name          = 'firecord'
   spec.version       = Firecord::VERSION
-  spec.authors       = ["Tomas Koutsky"]
-  spec.email         = ["tomas@stepnivlk.net"]
+  spec.authors       = ['Tomas Koutsky']
+  spec.email         = ['tomas@stepnivlk.net']
 
-  spec.summary       = %q{TODO: Write a short summary, because Rubygems requires one.}
-  spec.description   = %q{TODO: Write a longer description or delete this line.}
-  spec.homepage      = "TODO: Put your gem's website or public repo URL here."
-  spec.license       = "MIT"
+  spec.summary       = %q{Firecord is an ODM framework for Firebase in Ruby.}
+  spec.description   = %q{Firecord is an ODM (Object-Document-Mapper) framework for Firebase in Ruby.}
+  spec.homepage      = "http://stepnivlk.net"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.

[Read more about it here.](http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/StringLiterals)